src/server/scripts/Northrend/zone_sholazar_basin.cpp

```diff
@@ -800,8 +800,6 @@ class spell_q12589_shoot_rjr : public SpellScriptLoader
                     wilhelm->AI()->Talk(SAY_WILHELM_HIT);
                     if (Player* player = shooter->ToPlayer())
                         player->KilledMonsterCredit(NPC_APPLE);
-                    apple->DespawnOrUnsummon();
```

Comment:
Like said in #15472, if this unsummon happens, it will be immpossible to do the quest again, 
and, this unsummon is not blizzlike